shepherd_bms/include/compute.h

```diff
@@ -1,6 +1,23 @@
 #ifndef COMPUTE_H
 #define COMPUTE_H
 
-#include "datastructs.h"
+using namespace std;
 
+class ComputeInterface
+{
+    private:
+        int fanSpeed;
+
+    public: 
+        ComputeInterface();
+
+        ~ComputeInterface();
+
+        void enableCharging();
```

Comment:
enableCharging should get a boolean passed as a parameter called something like "isEnabled" or something. You might want to make a private boolean that we write this state to for now, and when we implement charging logic it'll become a bit more complex